templates/common/api-plugin-existing-api/teamsapp.yml.tpl

```diff
@@ -17,6 +17,19 @@ provision:
     writeToEnvironmentFile:
       teamsAppId: TEAMS_APP_ID
 
+{{#AuthType}}
```

Comment:
Seems it only check the authType whether exist, and how distinguish api key auth and oauth?

Reply:
Updated. Thanks!